### ISSUE TITLE
Cannot seem to draw to alt Graphics space

### ISSUE THREAD

**Issue by huadmin** [garciadelcastillo/-dashed-lines-for-processing-#2]
Using Creategraphics to write offscreen/alt canvas and then display doesn't seem to work?
I tried PG.dash.ellipse and dash.PG.ellipse.

Is this possible?

Thank you for the great tool

**Comment by heavyimage** [comment]
Just had the same issue.  Is there a workaround?  Thanks for such a nice library!

**Comment by garciadelcastillo** [comment]
Yes, this feature is currently untested and probably not working... :(

Help would be appreciated on this front!

**Comment by heavyimage** [comment]
If one could create a method to allow a client to pass in a new pgraphics context (currently g is protected) there would be an easy workaround, right?
